cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -474,15 +475,20 @@ func start(log log.Component,
 		}
 
 		webhooks, err := admissionpkg.StartControllers(admissionCtx, wmeta, pa)
-		if err != nil {
+		// Ignore the error if it's related to the missing validatingwebhookconfigurations RBACs
```

Comment:
Should the comment say that it's any error about validating webhooks instead of only RBACs?

Reply:
Yes 👍 I've changed the comment to: `// Ignore the error if it's related to the validatingwebhookconfigurations.`